app/Console/Commands/GDPRComplianceCommand.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use League\Csv\Reader;
+use Rogue\Models\Post;
+use Rogue\Services\AWS;
+use Rogue\Models\Signup;
+use Illuminate\Console\Command;
+use Rogue\Jobs\SendPostToQuasar;
+use Rogue\Jobs\SendSignupToQuasar;
+use Rogue\Jobs\SendPostToCustomerIo;
+use Rogue\Jobs\SendSignupToCustomerIo;
+
+class GDPRComplianceCommand extends Command
+{
+    /**
+     * AWS service class instance.
+     *
+     * @var \Rogue\Services\AWS
+     */
+    protected $aws;
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:gdpr {path}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Anonymizes data for users given in CSV in compliance with GDPR. Specifically, annonymizes caption and why_participated, deletes image URLs from s3 and tags all given users\' posts as hide-in-gallery.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @param AWS $aws
+     * @return void
+     */
+    public function __construct(AWS $aws)
+    {
+        parent::__construct();
+        $this->aws = $aws;
+    }
+
+    /**
+     * Executre the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        // Make a local copy of the CSV
+        $path = $this->argument('path');
+        $this->info('rogue:gdpr: Loading in csv from ' . $path);
+
+        $temp = tempnam('temp', 'command_csv');
+        file_put_contents($temp, fopen($this->argument('path'), 'r'));
+
+        // Load users from the CSV
+        $users_csv = Reader::createFromPath($temp, 'r');
+        $users_csv->setHeaderOffset(0);
+        $users = $users_csv->getRecords();
+
+        // Anonymize user data.
+        $this->info('rogue:GDPRcompliance: Anonymizing data...');
```

Comment:
Nitpicky, let's use `rogue:gdpr` here like our other log lines.

Reply:
ah yes - I shortened it and forgot to change there! Thanks for pointing that out!